File: Day_Four/javascript/TwoSumTest.js
const { getTwoSum } = require('./TwoSumTest');

test(" test That Method Returns The Two Number That Returns the Number That Makes Up The Sum ", ()=> {
        let array = {7, 6, 12, 4, -1};
        let sum = 6;
        let actual = TwoSumTest.getTwoSum(array, sum);
        let expected = {7, -1};
       expect(actual).toEqual(expected);

})

File: Day_Four/python/two_sum_test.py
from unittest import TestCase

import two_sum

class TestTwoSum(TestCase):

    def test_that_funtion_returns_the_two_number_thatMakes_up_its_sum(self):

        array = [8, 6, 12, 4, -2]
        number = 6
        actual = two_sum.get_two_Sum(array, number)
        expected = [8, -2]
        self.assertEqual(actual, expected)


class TestDuplicateElimination(TestCase):
    def     test_that_function_removes_dupliate_in_list_and_returns_the_modified_list(self):

        array = [9, 5, 1, 9, 4, 5, 1, 7]
        actual = two_sum.duplicate_elimination(array)
        expected = [9, 5, 1, 4, 7]
        self.assertEqual(actual, expected)
